tests/pipelines/pipeline_params.py

```diff
@@ -20,12 +20,6 @@
     ]
 )
 
-TEXT_TO_IMAGE_BATCH_PARAMS = frozenset(["prompt", "negative_prompt"])
```

Comment:
I just rearranged a bit, i.e. put all the batch inputs together, image inputs together 
did not delete or add anything